Tests/scripts/constants.py

```diff
@@ -250,6 +251,11 @@
 
 SCRIPTS_REGEX_LIST = [SCRIPT_YML_REGEX, SCRIPT_PY_REGEX, SCRIPT_JS_REGEX, SCRIPT_PS_REGEX]
 
+# All files that have related yml file
+REQUIRED_YML_FILE_TYPES = [SCRIPT_PY_REGEX, INTEGRATION_PY_REGEX, PACKS_INTEGRATION_PY_REGEX, PACKS_SCRIPT_PY_REGEX,
+                           SCRIPT_JS_REGEX, INTEGRATION_JS_REGEX, PACKS_SCRIPT_JS_REGEX, PACKS_INTEGRATION_JS_REGEX,
+                           PACKS_README_REGEX, INTEGRATION_README_REGEX]
```

Comment:
what about powershell? 😉

Reply:
I will leave it for when we start developing infra lol, didn't wanna mix
too much in the end ;)